examples/modules/txlistener.module.ts

```diff
@@ -32,9 +35,9 @@ export class TxlistenerApi extends API<Txlistener> implements Txlistener {
 // External class
 export class TxEmitter {

```

Comment:
In the current implementation (remix IDE code), I am not sure we need to have a `TxEmitter` . the current `TxListener` is listening either on new blocks or in a ethereumjs-vm event but it does not need to register directly to an emitter.
but that's a detail, we can merge this API def with the current implementation when it's time

Reply:
It's just an example of how a module can deal with an external class. In the end the implementation won't look exactly like that.